libavcodec/hevcdec.c

```diff
@@ -438,6 +438,9 @@ static enum AVPixelFormat get_format(HEVCContext *s, const HEVCSPS *sps)
     case AV_PIX_FMT_YUV444P12:
 #if CONFIG_HEVC_NVDEC_HWACCEL
         *fmt++ = AV_PIX_FMT_CUDA;
+#endif
+#if CONFIG_HEVC_VAAPI_HWACCEL
```

Comment:
It seems to be the decode support for 4:2:0 12bit, why the AV_PIX_FMT_VAAPI is enabled for YUV444P12?